health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/Svgicon.js

```
@@ -0,0 +1,14 @@
+import React from "react";
+
+export const Svgicon = () => (
+
+    <svg width="69" height="72" viewBox="0 0 69 72" fill="none" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)